tools/sgen/input/FluentUIStyle.yml

```diff
@@ -161,3 +163,21 @@ SecondaryButtonTokens extends MSFButtonTokens:
 GhostButtonTokens extends MSFButtonTokens:
   borderColor: [ rest: $Colors.Neutral.clear, hover: $Colors.Neutral.clear, pressed: $Colors.Neutral.clear, selected: $Colors.Brand.shade10, disabled: $Colors.Neutral.clear ]
 
+AP_MSFListTokens:
+  backgroundColor: [ rest: $Colors.Background.neutral1, pressed: $Colors.Background.neutral5 ]
+  borderColor: $Colors.Stroke.neutral1
+  borderSize: $Border.size.thin
+  disclosureIconForegroundColor: $Colors.Foreground.neutral3
+  disclosureInterspace: $Spacing.xxSmall
+  disclosureSize: $Icon.size.small
+  horizontalCellPadding: $Spacing.medium
+  iconColor: $Colors.Foreground.neutral4
+  iconInterspace: $Spacing.medium
+  iconSize: [ icon: $Icon.size.medium, largeIcon: $Icon.size.xxlarge ]
```

Comment:
Since the property is already named "iconSize", can we call we call the properties just with their predefined size name (small, medium, large...)?

Also, it seems like the largeIcon is not being used. What is the scenario you're planning to use it?

Reply:
Double-lined cells have the option of using large icons (as per figma)